Makefile

```diff
@@ -86,13 +86,16 @@ test:
 
 release: build/tinygo gen-device
 	@mkdir -p build/release/tinygo/bin
+	@mkdir -p build/release/tinygo/lib/clang/include
 	@mkdir -p build/release/tinygo/lib/CMSIS/CMSIS
 	@mkdir -p build/release/tinygo/lib/compiler-rt/lib
 	@mkdir -p build/release/tinygo/lib/nrfx
 	@mkdir -p build/release/tinygo/pkg/armv6m-none-eabi
 	@mkdir -p build/release/tinygo/pkg/armv7m-none-eabi
 	@mkdir -p build/release/tinygo/pkg/armv7em-none-eabi
+	@echo copying source files
 	@cp -p  build/tinygo                 build/release/tinygo/bin
+	@cp -p $(abspath $(CLANG_SRC))/lib/Headers/*.h build/release/tinygo/lib/clang/include
```

Comment:
And it is just that simple? Amazing.

Reply:
Well, there are a few generated header files (related to NEON and such) but I'm ignoring those for now. If the need ever arises, we can copy the generated header files.

Reply:
So this does only work for release tarballs, right? When using source with an external LLVM, I've taken to symlinking the header directory in, i.e.,
```
mkdir lib/clang
ln -s /usr/lib/clang/8.0.0/include lib/clang/include
```
I _think_ that's not documented anywhere?

Reply:
I think you're right, this should be added to the docs. Instead of symlinking, what if you do `make llvm-source` which downloads the LLVM source?

Reply:
Actually, this is a fixable bug. See #407 which should fix this, not requiring the symlink or checking out the LLVM sources.

Reply:
> `make llvm-source` which downloads the LLVM source?

Probably this would work, but I'd like to avoid it since I have the external compiled version installed and available already.

Reply:
That's what #407 avoids (which has now been merged into the dev branch).